src/lib/entropy/darwin_secrandom/info.txt

```diff
@@ -0,0 +1,17 @@
+define ENTROPY_SRC_DARWIN_SECRANDOM 20150925
+
+<source>
+darwin_secrandom.cpp
+</source>
+
+<header:internal>
+darwin_secrandom.h
+</header:internal>
+
+<os>
+darwin
+</os>
+
+<frameworks>
+darwin -> Security
```

Comment:
Yes, it must be added. This is similar to what is necessary when linking to a regular library with `-l`. Therefore, `./botan config libs` now returns `-lbotan-1.1 -framework Security` on my system.